compiler/lake.go

```diff
@@ -2,34 +2,45 @@ package compiler
 
 import (
 	"errors"
-	"runtime"
+	goruntime "runtime"
```

Comment:
Nit: When one of our packages clashes with the standard library like this, I think it'll be a little less surprising for other readers if we rename our package rather than the the standard library package.